### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Urs Behrmann

### DISCUSSION THREAD

**Discussion by UBehrmann** [heig-vd-dai-course/heig-vd-dai-course#249] [Show and tell]
https://github.com/UBehrmann

**Comment by ludelafo** [comment]
You seem to be pretty active on GitHub already! I've seen on your personal website that you have already coded your own web server in C++! Feel free to share your experience once we'll be at the second part of the course "Network programming with TCP and UDP". :)
